tests/testthat/test-process_content.R

```diff
@@ -12,6 +12,8 @@ test_that("processing content have proper output structure", {
   
   result$tokens |> is.numeric() |> expect_true()
   result$output |> is.list() |> expect_true()
+  result$total_nchar |> is.numeric() |> expect_true()
```

Comment:
tests fail here - should be probably `content_nchars` instead of `total_nchar`?
```suggestion
  result$content_nchars |> is.numeric() |> expect_true()
```